### DISCUSSION TITLE
Only Sync to specified graph

### DISCUSSION THREAD

**Discussion by Yersi88** [EINDEX/logseq-memos-sync#10] [Q&A]
I have two graphs in my logseq one for work notebook and one for personal notebook. I want Memos to only sync the memos to the personal graph. I don't want to have memos in work graph. How can I achieve this? It's very annoying every time I open work graph memos syncs it there, and I have to delete them.

**Comment by EINDEX** [comment]
In your working graph folder. the path is `assets/storages/logseq-memos-sync/syncStatus.json`.

You can change the lastSyncId value to something huge.

like: `{"lastSyncId":9999999}`

This will be a workaround to disable that.

**Comment by Yersi88** [comment]
Thank you, I tried. 

However, now it only synces when the 'Full sync' is enabled. And this is in the graph where I didn't touch syncStatus.json. 

Any idea why did this happen?
